include/thread_pool/mpmc_bounded_queue.hpp

```diff
@@ -169,7 +208,33 @@ inline MPMCBoundedQueue<T>& MPMCBoundedQueue<T>::operator=(MPMCBoundedQueue&& rh
 
```

Comment:
You don't need to use std::move with scalar types. m_buffer_mask is a scalar type you can simply copy it.

Reply:
Fixed in #3. Removed this altogether.